external-crates/move/crates/move-vm-types/src/values/value_prop_tests.rs

```diff
@@ -3,7 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use crate::values::{prop::layout_and_value_strategy, Value};
-use move_core_types::value::MoveValue;
+use move_core_types::runtime_value::MoveValue;
```

Comment:
How was this running before? 
@tzakian

Reply:
Looks like this wasn't caught before because this module is only turned on for fuzzing. So this change looks right and it's a good catch!